### DISCUSSION TITLE
Feature request: Longer history of park rating and guest count in the park management window

### DISCUSSION THREAD

**Discussion by MarcelVos96** [OpenRCT2/OpenRCT2#22609] [Feature requests]
Right now you can see the history of your park rating and guest count for up to a year.
![image](https://github.com/user-attachments/assets/58e1d974-666e-4a77-9048-bde3331e5b3b)

It would be fun if you could extend that window, like you already can when you're in the park overview tab, and see a much longer history.
![image](https://github.com/user-attachments/assets/a884b577-283e-4d09-9412-d99550989cbb)

The default size can stay the same, but then you can extend it yourself to see more.

**Comment by J-PIE-314** [comment]
PR #22414 has laid the groundwork for this to become a feature.

**Comment by J-PIE-314** [comment]
This feature request has been duplicated and expanded in #22686, and PR #22694 has been created to implement this feature.

**Reply by J-PIE-314** [reply]
PR https://github.com/OpenRCT2/OpenRCT2/pull/22694 has been merged.

One of your suggestions here is yet to be implemented ("see a much longer history").
